Core/DISET/private/Transports/SSL/SocketInfoFactory.py

```diff
@@ -57,7 +57,7 @@ def __sockConnect( self, hostAddress, sockType, timeout, retries ):
     osSocket = socket.socket( sockType, socket.SOCK_STREAM )
     #osSocket.setblocking( 0 )
     if timeout:
-      osSocket.settimeout( 5 )
+      osSocket.settimeout( 0.5 ) # we retry 3 times, which is 3 sec + 5 times retry = 15 sec.
```

Comment:
Will 0.5 be OK for slow networks ? The total period of retries looks OK ( 15 sec ), this is in most cases enough for a service to restart. But may be it is better to have a slightly longer timeout but less number of retries.